cabal-install/src/Distribution/Client/Version.hs

```diff
@@ -5,11 +5,13 @@ module Distribution.Client.Version
 
 import Distribution.Version
 
+import qualified Paths_cabal_install as PackageInfo
+
 -- This value determines the `cabal-install --version` output.
 --
 -- It is used in several places throughout the project, including anonymous build reports, client configuration,
 -- and project planning output. Historically, this was a @Paths_*@ module, however, this conflicted with
```

Comment:
This comment seems a bit worrying - are we sure this doesn't resurrect that issue? Or is it fine to _delegate_ to `Paths_`?

Reply:
This comment should probably be removed, I don't think we are still using HPC (or that HPC is still usable).

Reply:
I really wonder what the conflict with HPC was... That said, I have no specific advice here.

As a maintainer of HPC related tooling I have to slightly protest that it isn't usable :) I am also working on better integration of hpc with other tools such as cabal, e.g. https://github.com/ghc-proposals/ghc-proposals/pull/612 which intends to improve the way that it can be integrated into external tooling.

Reply:
Glad to have you in the team then :) Let's move on with this change and see what happens with HPC?